src/DesignKit.tsx

```diff
@@ -2987,6 +2988,234 @@ const MessageSection: React.FC = () => {
+          The Elevation module is a set of CSS variables and classes that serves the purpose of
+          highlighting levels of interaction in the application (like Cards, context menus, etc).
+        </Paragraph>
+      </AntDCard>
+      <AntDCard title="Usage">
+        <Paragraph>
+          To begin with, the Elevation module can be used with the CSS variables only, there are
+          variables for each aspect of the elevation module:
+        </Paragraph>
+        <br />
+        <Paragraph>
+          Shadow, which has to be used only on a set of special elements like cards and tooltips
+          (small); menu and toasts (medium); and modals (large) (to be used in the{' '}
+          <code>{'box-shadow'}</code> CSS property):
+        </Paragraph>
+        <Columns>
+          <code>{'--elevation-shadow-small'}</code>
+          <code>{'--elevation-shadow-medium'}</code>
+          <code>{'--elevation-shadow-large'}</code>
+        </Columns>
+        <br />
+        <Paragraph>
+          Background elevation (<code>{'elevation-bg'}</code>), which is used to depict the
+          elevation itself, and to hilight some interactions.
+        </Paragraph>
+        <Paragraph>
+          The base (<code>{'elevation-bg-base'}</code>) should be used in all elements, where, for
+          some special cases where is necessary to have some contrast, it should add the{' '}
+          <code>{'elevation-bg-strong'}</code> on top of that.
+        </Paragraph>
+        <Paragraph>
+          The (<code>{'elevation-bg-contrast'}</code>) should be used in a few special tooltips.
+        </Paragraph>
+        <Paragraph>
+          The (<code>{'elevation-bg-backdrop'}</code>) should be used in any backdrop element.
+        </Paragraph>
+        <Paragraph>
+          The (<code>{'elevation-bg-border'}</code>) should be used in all level 1 elevation
+          components (except cards).
+        </Paragraph>
+        <Paragraph>
+          The (<code>{'elevation-bg-focused'}</code>) should be used in all level 1 elevation
+          components that are being focused (except cards).
+        </Paragraph>
+        <Columns>
+          <code>{'--elevation-bg-base-light'}</code>
+          <code>{'--elevation-bg-base-dark'}</code>
+          <code>{'--elevation-bg-strong-light'}</code>
+          <code>{'--elevation-bg-strong-dark'}</code>
+          <code>{'--elevation-bg-contrast-light'}</code>
+          <code>{'--elevation-bg-contrast-dark'}</code>
+          <code>{'--elevation-bg-backdrop'}</code>
+        </Columns>
+        <Columns>
+          <code>{'--elevation-bg-selected-light'}</code>
+          <code>{'--elevation-bg-selected-dark'}</code>
+          <code>{'--elevation-bg-hover-light'}</code>
+          <code>{'--elevation-bg-hover-dark'}</code>
+          <code>{'--elevation-bg-focused'}</code>
+          <code>{'--elevation-bg-border-light'}</code>
+          <code>{'--elevation-bg-border-dark'}</code>
+        </Columns>
```

Comment:
fix: these should just be in one container that wraps, like a Space. This applies to the other places you did this as well. Also, is it necessary to include the `{' ... '}` around the text?